lib/run/index.js

```diff
@@ -158,19 +158,23 @@ module.exports = function (options, callback) {
                         summary: emitter.summary
                     });
 
-                    _.forEach(['environment', 'globals', 'collection'], function (item) {
+                    var ENV_EXPORT_KEYS = ['id', 'name', 'synced', 'syncedFilename'];
```

Comment:
What's these two - synced and syncedFilename? If these are app export quirks, they should not be here + @czardoz